### PR TITLE
Add Homarr

### PR DESCRIPTION
**Pull request by meonkeys**
- [x] Submit one item per pull request. This eases reviewing and speeds up inclusion.
- [x] Format your submission as follows, where `Demo` and `Clients` are optional.
  - Do not add a duplicate `Source code` link if it is the same as the main link.
  - Keep the short description under 250 characters and use [sentence case](https://en.wikipedia.org/wiki/Letter_case#Sentence_case) for it, even if the project's webpage or readme uses another capitalisation such as title case, all caps, small caps or all lowercase.
  - `Demo` links should only be used for interactive demos, i.e. not video demonstrations.
  - ``- [Name](http://homepage/) - Short description, under 250 characters, sentence case. ([Demo](http://url.to/demo), [Source Code](http://url.of/source/code), [Clients](https://url.to/list/of/related/clients-or-apps)) `License` `Language` ``
- [x] Additions that depend on proprietary services outside the user's control must be marked `⚠`.
  - ``- [Name](http://homepage/) `⚠` - Short description, under 250 characters, sentence case. ([Demo](http://url.to/demo), [Source Code](http://url.of/source/code), [Clients](https://url.to/list/of/related/clients-or-apps)) `License` `Language` ``
- [x] Additions that are not [Free software](https://en.wikipedia.org/wiki/Free_software) must be added to `non-free.md` and have the license set to `⊘ Proprietary`
- [x] Additions are inserted preserving alphabetical order.
- [x] Additions are not already listed at any of [awesome-sysadmin](https://github.com/n1trux/awesome-sysadmin), [staticgen.com](https://www.staticgen.com/), [staticsitegenerators.net](https://staticsitegenerators.net/).
- [x] The `Language` tag is the main **server-side** requirement for the software. Don't include frameworks or specific dialects.
- [x] Any license you add is in our [list of licenses](https://github.com/awesome-selfhosted/awesome-selfhosted/blob/master/README.md#list-of-licenses).
- [x] You have searched the repository for any relevant [issues](https://github.com/awesome-selfhosted/awesome-selfhosted/issues) or [PRs](https://github.com/awesome-selfhosted/awesome-selfhosted/pulls), including closed ones.
- [x] Any category you are creating has the minimum requirement of 3 items.
  - If not, your addition may be inserted into `Misc/Other`.
- [x] Any software project you are adding to the list is actively maintained.
- [x] Any software project you are adding has working installation instructions.
- [x] The pull request title is informative, unlike "Update README.md".
  - Suggested titles: "Add aaa to bbb" for adding software aaa to section bbb,
  - "Remove aaa from bbb" for removing, "Fix license for aaa", etc.

### CODE REVIEW

**Review by Lissy93 — APPROVED** [review]

**Review by Lissy93 — CHANGES_REQUESTED** [review]
I believe the technologies denotation should be `Nodejs/Docker`


> The Language tag is the main server-side requirement for the software - don't include frameworks or specific dialects.

**Comment by meonkeys** [comment]
Oh yeah, thanks for bringing that up! I read that in the PR template, got confused, and took a guess. I picked "Docker" because, for Homarr, [Docker _is_ the one recommended server-side requirement](https://homarr.dev/docs/introduction/installation). I'm sure there's some way to spin up your own MEAN-like stack but I couldn't find it after a quick scan of their docks.

I'm also seeing other apps on the awesome-selfhosted list with "Language" as `Go/Docker`, `Docker/K8S/Go/Nodejs`, and `Python/Nodejs/Docker`.

I'm good either way, I just wasn't sure what the convention/intent is here.

File: README.md [hunk 1]
```diff
@@ -1547,6 +1547,7 @@ _Related: [Monitoring](#monitoring), [Bookmarks and Link Sharing](#bookmarks-and
 - [Hiccup](https://designedbyashw.in/test/hiccup/) - A beautiful static homepage to get to your links and services quickly. It has built-in search, editing, PWA support and localstorage caching to easily organize your start page. ([Source Code](https://github.com/ashwin-pc/hiccup)) `MIT` `HTML5`
 - [Homepage by benphelps](https://github.com/benphelps/homepage) - A highly customizable homepage (or startpage / application dashboard) with Docker and service API integrations. `GPL-3.0` `Docker/Nodejs`
 - [Homepage by tomershvueli](https://github.com/tomershvueli/homepage) - Simple, standalone, self-hosted PHP page that is your window to your server and the web. `MIT` `PHP`
+- [Homarr](https://homarr.dev) - Sleek, modern dashboard with many integrations and web-based config. ([Demo](https://demo.homarr.dev), [Source Code](https://github.com/ajnart/homarr)) `MIT` `Docker`
```
Comment (nodiscc) [inline README.md:1550]:
```suggestion
- [Homarr](https://homarr.dev) - Sleek, modern dashboard with many integrations and web-based config. ([Demo](https://demo.homarr.dev), [Source Code](https://github.com/ajnart/homarr)) `MIT` `Docker/Nodejs`
```

> I just wasn't sure what the convention/intent is here.

The documented installation methods are Docker, or Nodejs (from source/`yarn`), in that order.

**Review by nodiscc — CHANGES_REQUESTED** [review]

**Comment by meonkeys** [comment]
I think I'm done from my end. LMK if it is good to go or needs more work.

**Review by nodiscc — APPROVED** [review]

**Review by Lissy93 — APPROVED** [review]

**Pull request merged**